bitcoinspv/clients/sui/sui_spv_client.go

```diff
@@ -92,167 +126,141 @@ func (c *SPVClient) InsertHeaders(ctx context.Context, blockHeaders []wire.Block
 }
 
 // ContainsBlock checks if the light client's chain includes a block with the given hash.
-func (c *SPVClient) ContainsBlock(ctx context.Context, blockHash chainhash.Hash) (bool, error) {
-	arguments := []interface{}{
-		c.lcObjectID,
-		BlockHashToHex(blockHash),
-	}
+func (c *BTCLightClientObject) ContainsBlock(ctx context.Context, blockHash chainhash.Hash) (bool, error) {
+	ptb := suiptb.NewTransactionDataTransactionBuilder()
 
-	response, err := c.moveCall(ctx, containsBlockFunc, arguments)
+	b, err := bcs.Marshal(blockHash[:])
```

Comment:
```suggestion
	blockBCS, err := bcs.Marshal(blockHash[:])
```

Reply:
or `blockEncoded`?

Reply:
I follow the short name convention in go